modules/react/common/stories/Responsive.stories.mdx

```diff
@@ -0,0 +1,333 @@
+import {Meta, ArgsTable} from '@storybook/addon-docs';
+import {Box} from '@workday/canvas-kit-react/layout';
+//
+import {
+  useResizeObserver,
+  ResponsiveContextProvider,
+  useResponsiveStyles,
+} from '@workday/canvas-kit-react/common';
+import {ResponsiveContainer} from './examples/ResponsiveStyles';
+import {ResponsiveViewport} from './examples/ResponsiveViewport';
+
+<Meta title="Hooks and Utilities/Responsive Styling" />
+
+# Responsive Styling
+
+Responsive styling is an approach to web development or a set of best practices, used to create a
+layout that can respond to the device being used to view the content.
+
+You can read more about the history of
+[Responsive Design](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Responsive_Design).
+
+## Container and Viewport Based Responsive Styling
+
+We will take a look at Container-based and Viewport-based Responsive Styling, what they are and
+which to use.
+
+### Container Based Responsive Styling
+
+Now, with viewport-based responsive styling (media queries), the user's viewport (user's visible
+area of a web page) is measured and styles are applied based on the size of that viewport. We will
+do the same with Container Based Responsive Styling but, we will just be measuring the size of the
+container the element is in instead of the user's viewport.
+
+Take the below example of a component:
+
+```tsx
+<Box maxWidth={1000}>Hello World</Box>
+```
+
+In the example above, we see that there is a `maxWidth` set on the Box component of `1000`. This
+will set the maximum width of that element at `1000px`. What we can do with Container Based
+Responsive Styling is measure when that container's width reaches certain breakpoints
+`(i.e. 1440px, 1024px, 768px, 320px, 0)` and apply styles based on those breakpoints. If the
+container's width were to reach `768px`, we can apply styles to that breakpoint.
+
+In the interactive example below, when the dropdown's value is updated, the container size will
```

Comment:
in the above paragraph you mention an example above and in the bottom paragraph you mention an example below. Maybe we can just describe what is happening in one section and let the example describe what's happening. We have a similar patterns in our MDX where we describe some storybook example and then get the example below.

Reply:
Definitely agree, I will update this